src/components/Bag/downloads/index.js

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+
+import {
+    Button,
+    Dialog,
+    DialogActions,
+    DialogContent,
+    DialogTitle,
+    IconButton,
+    List,
+    ListItem,
+    makeStyles,
+    Typography,
+} from "@material-ui/core";
+
+import { Cancel as CancelIcon, Close } from "@material-ui/icons";
+
+import { build as buildID } from "@cyverse-de/ui-lib";
+
+import { useTranslation } from "i18n";
+
+import { getHost } from "components/utils/getHost";
+import constants from "components/Bag/constants";
+
+const useStyles = makeStyles((theme) => ({
+    closeButton: {
+        position: "absolute",
+        right: theme.spacing(1),
+        top: theme.spacing(1),
+    },
+    help: {
+        paddingTop: theme.spacing(2),
+        paddingBottom: theme.spacing(1),
+    },
+    paper: {
+        width: theme.spacing(60),
+        height: theme.spacing(70),
+    },
+}));
+
+const DownloadLinksDialog = ({
+    paths = [],
+    open = false,
+    fullScreen = false,
+    onClose = () => {},
+}) => {
+    const classes = useStyles();
+    const { t } = useTranslation(["bags", "common"]);
+    const dialogID = buildID(constants.DOWNLOAD_BASEID, constants.DIALOG);
+
+    const handleClose = (event) => {
+        event.preventDefault();
+        event.stopPropagation();
+
+        onClose();
+    };
+
+    return (
+        <Dialog
+            fullScreen={fullScreen}
+            open={open}
+            onClose={handleClose}
+            classes={{ paper: classes.paper }}
```

Comment:
It'd be good if the dialog itself had its own static ID as well, looks like it should be `dialogID` from what I can tell

Reply:
This is going into a separate PR.